docs/02_Hardware_References/01_Full_specs.md

```diff
@@ -0,0 +1,78 @@
+
+# Full Specifications
+
+Below the full technical specifications
+
+| | **ESP32** | **RP2040** |
+| --- | --- | --- |
+| *Memory* | 8 MB PSRAM | 264 KB PSRAM |
+| *Flash storage* | 16 MB internal SPI flash | 64M-bit external QSPI flash |
+| *Connectivity* | Wi-Fi 802.11 b/g/n (802.11n up to 150 Mbps),  Bluetooth v4.2 BR/EDR and BLE specification | - |
+| *RF power* | WiFi: 20 dBm, BT:13.7 dBM | - |
+| *Modulation type* | GFSK, OFDM | - |
+| *Frequency bands* | 2400 ÷ 2483.5 MHz | - |
+| *Bandwidth* | BT / BLE: 1 MHz (79 channels) / 2 MHz (40 channels) | - |
+| *Peripherals* | UART, I²C, SPI on UEXT connector, blue and yellow LEDS, SWD lines to RP2040 | UART, SPI, I²C on P1 and P2 , green LED |
+| *Sensors* | SPK0838HT4H-1 Microphone, shared ICM-20948 IMU | Shared ICM-20948 IMU |
+
+</p>
+
+*Operating temperature*: 0°C ÷ +60°C  
+*Power supply*: +5VDC  
+*Dimensions*: 130 x 40 x 10.9 mm
+
+## Resources
+
+In addition to this guide, several ones are available online:
++ for a full explanation of the UDOO KEY’s hardware download the [UDOO KEY User Manual](https://udoo.org/download/files/UDOO_KEY/Doc/UDOO_KEY_MANUAL.pdf)
++ see the [Get Started - ESP32](/03_Get_Started/00_Get_started_with_ESP32.md) and [Get Started - RP2040](/03_Get_Started/00_Get_started_with_RP2040.md) sections for all the useful information for a quick installation
++ [here](https://udoo.org/download/files/UDOO_KEY/Doc/UDOO_KEY_SCHEMATICS.pdf) you can find the schematics file for the UDOO KEY
++ [official ESP32 development framework](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/index.html)
++ [Raspberry PI PICO datasheet](https://datasheets.raspberrypi.com/pico/pico-datasheet.pdf)
++ [Raspberry PI PICO C/C++ SDK](https://datasheets.raspberrypi.com/pico/raspberry-pi-pico-c-sdk.pdf)
++ [MicroPython download page for RP2040](https://micropython.org/download/rp2-pico/)
+
+## Board versions
+
+The UDOO KEY is available in two versions: **UDOO KEY Basic** and **UDOO KEY Pro**.  
+The only difference between the two is whether or not the board is equipped with sensors.
```

Comment:
Drop this line

Reply:
Solved with bbe86fa1935fee4666925dabb13f68beaf635452